README.md

```diff
@@ -36,6 +36,17 @@ producing complex, multi-container applications.
    * `docker` is default service and does not require above configuration.
    * Enable multiple services as comma separated list. Eg: 'docker, openshift'
 
+4. Enable any specific options for the services you have selected:
+
+    * OpenShift
+
+        Specific versions can be specified using the following variables:
+
+        `config.servicemanager.openshift.docker_registry = "docker.io"` - What registry should be pulled from
```

Comment:
@navidshaikh Not quite sure whether the README makes a good TODO. If I visit the repo and read the readme, I would expect the things mentioned there to be implemented. Maybe not releases, but I'd install from source I'd expect it to work. IMO either use github issues or wiki page or utilize the TODO file which already exists.